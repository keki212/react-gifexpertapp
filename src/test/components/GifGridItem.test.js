import React from 'react'
import { shallow } from 'enzyme';
import GifGridItem from '../../components/GifGridItem';

describe('Prueba del componente <GifGridItem />', () => {
    const title = 'Imagen para la prueba';
    const url ='https://imagenes.com/images.jpg';
    
    test('Debe mostrar el componente correctamente', () => {
        const wrapper = shallow(<GifGridItem title={title} url={url} />)
        expect(wrapper).toMatchSnapshot();
    })
    
})
