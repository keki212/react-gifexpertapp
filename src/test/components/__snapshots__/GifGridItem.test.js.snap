// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba del componente <GifGridItem /> Debe mostrar el componente correctamente 1`] = `
<div
  className="card animate__animated animate__backInRight"
>
  <img
    alt="Imagen para la prueba"
    src="https://imagenes.com/images.jpg"
  />
  <p>
    Imagen para la prueba
  </p>
</div>
`;
